### PR TITLE
Add commands to generate staking address keys and stake pool keys

### PR DESCRIPTION
**Pull request by Jimbo4350**
Issue
-----------

- This PR **does not result** in breaking changes to upstream dependencies.

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Run.hs [hunk 1]
```diff
@@ -187,6 +190,20 @@ runNodeIssueOpCert (VerificationKeyFile vkeyKESPath)
...
 
+runNodeStakingKeyGen :: VerificationKeyFile -> SigningKeyFile -> ExceptT CliError IO ()
+runNodeStakingKeyGen (VerificationKeyFile vkFp) (SigningKeyFile skFp) = do
+  (vkey, skey) <- liftIO genKeyPair
+  firstExceptT CardanoApiError . newExceptT $ writeVerificationKeyStaking vkFp vkey
+  --TODO: writeSigningKey should really come from Cardano.Config.Shelley.ColdKeys
+  firstExceptT CardanoApiError . newExceptT $ writeSigningKey skFp (SigningKeyShelley skey)
+
+runNodeStakePoolGeyGen :: VerificationKeyFile -> SigningKeyFile -> ExceptT CliError IO ()
+runNodeStakePoolGeyGen (VerificationKeyFile vkFp) (SigningKeyFile skFp) = do
```
Comment (disassembler) [inline cardano-cli/src/Cardano/CLI/Shelley/Run.hs:201]:
```suggestion
runNodeStakePoolKeyGen :: VerificationKeyFile -> SigningKeyFile -> ExceptT CliError IO ()
runNodeStakePoolKeyGen (VerificationKeyFile vkFp) (SigningKeyFile skFp) = do
```

File: cardano-cli/src/Cardano/CLI/Shelley/Run.hs [hunk 2]
```diff
@@ -83,6 +84,8 @@ runNodeCmd (NodeKeyGenKES  vk sk)     = runNodeKeyGenKES  vk sk
 runNodeCmd (NodeKeyGenVRF  vk sk)     = runNodeKeyGenVRF  vk sk
 runNodeCmd (NodeIssueOpCert vk sk ctr p out) =
   runNodeIssueOpCert vk sk ctr p out
+runNodeCmd (NodeStakingKeyGen vk sk) = runNodeStakingKeyGen vk sk
+runNodeCmd (NodeStakePoolKeyGen vk sk) = runNodeStakePoolGeyGen vk sk
```
[the remark this answers is not in the document]
Comment (erikd) [reply cardano-cli/src/Cardano/CLI/Shelley/Run.hs]: Fixed.

File: cardano-cli/src/Cardano/CLI/Shelley/Run.hs (hunk 1 above)
Comment (erikd) [reply cardano-cli/src/Cardano/CLI/Shelley/Run.hs]: Fixed.

**Pull request merged**